Dockerfile

```diff
@@ -93,6 +96,7 @@ COPY --from=server_builder /go/src/github.com/spacemeshos/go-spacemesh/build/go-
 COPY --from=server_builder /go/src/github.com/spacemeshos/go-spacemesh/build/go-p2p /bin/
 COPY --from=server_builder /go/src/github.com/spacemeshos/go-spacemesh/build/go-harness /bin/
 COPY --from=server_builder /go/src/github.com/spacemeshos/go-spacemesh/build/libgpu-setup.so /bin/
+COPY --from=server_builder /go/src/github.com/spacemeshos/go-spacemesh/build/libsvm.so /bin/
```

Comment:
is it large? not sure if we should add it to the image given that it won't be used in near future (6m at least?)

Reply:
13,7 MB

Reply:
I can comment it out until we need it
cc @moshababo

Reply:
Yes, plz comment it out for now.